src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+@import "./utils/_functions.scss";
```

Comment:
why do you need to import it here if you import it in the main file?

Reply:
I just left it as a reminder (in case if the 'stars' block is further used as a separate block, it reminds that there is also a 'function' which applies to it).

Reply:
No need to save, something for the future. First, git has versions and you will not lose the code that is important to you, secondly, business tasks can change and unused code may never be used, but it can interfere and disorient.